setup.py

```diff
@@ -6,7 +6,7 @@
 # Get the long description from the README file
 LONG_DESCRIPTION = (here / "README.md").read_text(encoding="utf-8")
 
-VERSION = "1.19.0"
+VERSION = "1.20.0"
```

Comment:
You don't need to change this, it will automatically get updated and published once this PR merges.